app.css

```diff
@@ -39,6 +39,14 @@ footer {
   margin-bottom: 2rem;
 }
 
+footer .bigger-link {
```

Comment:
What if you just take the `0.8em` off of `footer` instead of adding another version of this class?

Reply:
ok, tried to simplify some, but needed to wrap each line in a div for proper spacing